src/AppSystem/Launcher.vala

```diff
@@ -377,6 +378,19 @@ public class Dock.Launcher : BaseItem {
         });
     }
 
+    private void update_tooltip () {
+        string[] accels = {};
+        var index = (int) current_pos / ItemManager.get_launcher_size ();
+        if (index < 9) {
+            accels = {"<Super>%i".printf (index + 1)};
```

Comment:
The shortcuts are configurable so we probably should make sure to use the actually configured one here